File: crud-dotnet-api/crud-dotnet-api/ClientApp/src/app/common/note.model.ts
import { FormBuilder, FormControl, FormGroup } from "@angular/forms";

export class NoteModel{
    UserId:string="";
    Label:string="";
    Titile:string="";
    Content:string="";
    BackgroundColor:string="";
    FontColor:string="";  
    NoteRegistrationForm:FormGroup;

    constructor(){
        let formBuilder = new FormBuilder();
        this.NoteRegistrationForm = formBuilder.group({
         Label: new FormControl(''),
         Titile: new FormControl(''),
         Content: new FormControl(''),
         BackgroundColor: new FormControl(''),
         FontColor: new FormControl('')
        })
    }
}

File: crud-dotnet-api/crud-dotnet-api/Model/NoteModel.cs
namespace crud_dotnet_api.Model
{
    public class NoteModel
    {
        public Guid Id { get; set; }
        public string Title { get; set; }
        public string Label { get; set; }
        public string Content { get; set; }
        public string BackgroundColor { get; set;}
        public string FontColor { get; set; }
        public DateTime CreatedOn { get; set; }
        public DateTime ModifiedOn { get; set; }
        public ApplicationUser User { get; set; }   
        public Guid UserId { get; set; }

    }
}
